translations_to_weblate.sh

```diff
@@ -1,23 +1,15 @@
 #!/bin/bash
-target='*.tsl'
-output='BE_weblate.json'
+MY_PATH="`dirname \"$0\"`"
```

Comment:
I think you don't need to escape the doublequotes here, unless they are part of the name ;)